src/deluge/gui/menu_item/toggle.h

```diff
@@ -14,4 +14,17 @@ class Toggle : public Value<bool> {
 	virtual void drawValue();
 	void drawPixelsForOled();
 };
+
+class ToggleBool : public Toggle {
+public:
+	using Toggle::Toggle;
+
+	ToggleBool(l10n::String newName, l10n::String title, bool& newToggle) : Toggle(newName, title), t(newToggle) {}
```

Comment:
This should have a comment on the bool pointers lifetime - in this case it must last at least as long as the menu exists, and so can't be valid for song specific toggles

Reply:
Added thanks!